protoc-gen-go/testdata/Makefile

```diff
@@ -57,7 +57,7 @@ testbuild:	regenerate
 
 regenerate:
 	# Invoke protoc once to generate three independent .pb.go files in the same package.
-	protoc --go_out=. multi/multi{1,2,3}.proto
```

Comment:
Make didn't pick up the right shell on my dev VM, hence globbing failed. So I rewrote this to reduce the dependency on shell.